src/libs/actions/TransactionEdit.ts

```diff
@@ -39,4 +39,21 @@ function restoreOriginalTransactionFromBackup(transactionID: string, isDraft: bo
     });
 }
 
-export {createBackupTransaction, removeBackupTransaction, restoreOriginalTransactionFromBackup};
+function createDraftTransaction(transaction: OnyxEntry<Transaction>) {
+    if (!transaction) {
+        return;
+    }
+
+    const newTransaction = {
+        ...transaction,
+    };
```

Comment:
Why do we have to d othis if we make no changes to the object? @DylanDylann @ikevin127

Reply:
Missed that spread, and you're right, I'd pass the transaction directly unless there's a specific reason to spread it (cloning, immutability, future plan to extend it) ? 🤔 

It might've been copy & pasted from here where it's the same:

https://github.com/Expensify/App/blob/77a3c12a590d539ebd136443190451bc20e922fa/src/libs/actions/IOU.ts#L314-L324